scipy/stats/_stats_py.py

```diff
@@ -4642,28 +4642,83 @@ def fisher_exact(table, alternative='two-sided'):
     conditional maximum likelihood estimate of the odds ratio, use
     `scipy.stats.contingency.odds_ratio`.
 
+    References
+    ----------
+    .. [1] Fisher, Sir Ronald A, "The Design of Experiments:
+           Mathematics of a Lady Tasting Tea." ISBN 978-0-486-41151-4, 1935.
+    .. [2] "Fisher's exact test",
+           https://en.wikipedia.org/wiki/Fisher's_exact_test
+    .. [3] Emma V. Low et al. "Identifying the lowest effective dose of
+           acetazolamide for the prophylaxis of acute mountain sickness:
+           systematic review and meta-analysis."
+           BMJ, 345, :doi:`10.1136/bmj.e6779`, 2012.
+
     Examples
     --------
-    Say we spend a few days counting whales and sharks in the Atlantic and
-    Indian oceans. In the Atlantic ocean we find 8 whales and 1 shark, in the
-    Indian ocean 2 whales and 5 sharks. Then our contingency table is::
+    In [3]_, the effective dose of acetazolamide for the prophylaxis of acute
+    mountain sickness was investigated. The study notably concluded:
+
+        Acetazolamide 250 mg, 500 mg, and 750 mg daily were all efficacious for
+        preventing acute mountain sickness. Acetazolamide 250 mg was the lowest
+        effective dose with available evidence for this indication.
+
+    Identifying the correct effective dose is particularly important as
+    side effects to acetazolamide are frequent.
+
+    The following table summarizes the results of the experiment in which
+    some participants took a daily dose of acetazolamide 250 mg while others
+    took a placebo.
+    Cases of acute montain sickness were recorded::
+
+                                    Acetazolamide   Control/Placebo
+        Acute montain sickness             7           17
+        No                                15            5
+
+    Is there evidence that the acetazolamide 250 mg reduces the risk of
+    acute montain sickness?
+    We begin by formulating a null hypothesis :math:`H_0`:
 
-                Atlantic  Indian
-        whales     8        2
-        sharks     1        5
+        The effect of acetazolamide is equivalent to that of placebo.
 
-    We use this table to find the p-value:
+    Let's assess the plausibility of this hypothesis with
+    Fisher's test.
 
     >>> from scipy.stats import fisher_exact
-    >>> res = fisher_exact([[8, 2], [1, 5]])
+    >>> res = fisher_exact([[7, 17], [15, 5]])
```

Comment:
```suggestion
    >>> res = fisher_exact([[7, 17], [15, 5]], alternative='less')
```
It seems that it is appropriate to use the `alternative` parameter here since it's available (unlike `chi2_contingency`) and we want the alternative hypothesis to be that the true odds ratio is `"less"` than 1 (the true odds of getting accute mountain sickness with acetazolamide is less than the true odds of getting accute mountain sickness with placebo).

Did they not do this in the paper?

Reply:
Seems better indeed 👍 Shall I add a sentence to phrase the nuance of using this parameter?

In the paper they only speak in terms of odd-ratio, they don't calculate Fisher's test.